### ISSUE TITLE
Can't compile appsignal due to checksum verification fail

### ISSUE THREAD

**Issue by schurig** [appsignal/appsignal-elixir#15]
I added `{:appsignal, "~> 0.3"}` to my mix.exs

When compiling the deps I get the following error

```
==> appsignal
could not compile dependency :appsignal, "mix compile" failed. You can recompile this dependency with "mix deps.compile appsignal", update it with "mix deps.update appsignal" or clean it with "mix deps.clean appsignal"
==> my_app
** (Mix) Checksum verification of /tmp/appsignal-agent-220a888.tar.gz failed!
Calculated: 737dbd50b92e6f105de0e11aaab5de9bc2404ed999e341fc7822bd5d71205b5d
  Expected: 5e313156339389b339c289592327ac1c3d57999df93cd31ce9c69fa4ede1e38f
```

**Comment by schurig** [comment]
Solved it with #5

**Comment by arjan** [comment]
Always love issues that solve themselves ;-)
